src/request.ts

```diff
@@ -11,9 +11,12 @@ const newRequestFromIncoming = (
   incoming: IncomingMessage | Http2ServerRequest
 ): Request => {
   const headerRecord: [string, string][] = []
-  const len = incoming.rawHeaders.length
-  for (let i = 0; i < len; i += 2) {
-    headerRecord.push([incoming.rawHeaders[i], incoming.rawHeaders[i + 1]])
+  const rawHeaders = incoming.rawHeaders
+  for (let i = 0; i < rawHeaders.length; i += 2) {
+    const {[i]: key, [i + 1]: value} = rawHeaders
+    if (key.charCodeAt(0) !== /*:*/ 0x3a) {
```

Comment:
Undici had a bug that allowed ':', which was not originally allowed in the header value, but now that it is fixed, this is done.

Reply:
Undici had a bug that allowed ':', which was not allowed in the header key, but now that it is fixed.